src/shapes/text.class.js

```diff
@@ -1343,13 +1343,13 @@
      * @return {Number} Line width
      */
     getLineWidth: function(lineIndex) {
-      if (this.__lineWidths[lineIndex]) {
+      if (this.__lineWidths[lineIndex] !== undefined) {
```

Comment:
Avoid causing a cache miss when the cached value is zero

Reply:
this is fine